libkbfs/bserver_remote.go

```diff
@@ -533,7 +533,15 @@ func (b *BlockServerRemote) GetUserQuotaInfo(ctx context.Context) (info *kbfsblo
 		b.log.LazyTrace(ctx, "BServer: GetUserQuotaInfo done (err=%v)", err)
 	}()
 	res, err := b.getConn.getClient().GetUserQuotaInfo(ctx)
-	return kbfsblock.ParseGetQuotaInfoRes(b.config.Codec(), res, err)
+	info, err = kbfsblock.ParseGetQuotaInfoRes(b.config.Codec(), res, err)
+	if err != nil {
+		return nil, err
+	}
+	for f, us := range info.Folders {
+		b.log.CDebugf(ctx, "Folder %s, archivedBytes=%d, gitArchiveBytes=%d",
+			f, us.Bytes[kbfsblock.UsageArchive], us.Bytes[kbfsblock.UsageGitArchive])
```

Comment:
Do we really want to be logging this on every `GetUserQuotaInfo`?

Reply:
Oops, no we don't.  This was for my personal debugging.  Will remove, thanks for catching!